### ISSUE TITLE
Modify security group?

### ISSUE THREAD

**Issue by osrf-migration** [osrf/cloudsim-legacy#38]
**Original report ([archived issue](https://osrf-migration.github.io/osrf-archived-gh-pages/#!/osrf/cloudsim-legacy/issues/38)) by Chad Jenkins (Bitbucket: [odestcj](https://bitbucket.org/%7B5e503a35-1f78-48c6-9010-edbd488b5cdc%7D/)).**

----------------------------------------

Is there a reasonable way to modify the security groups for simulator instances?  If not, I would like to lobby for 8080 and 9090 to be included in the open ports.

**Comment by osrf-migration** [comment]
**Original comment by Stephen Brawner (Bitbucket: [brawner](https://bitbucket.org/%7Bf6bd3fe7-4d17-4503-9232-0915d6e97879%7D/)).**

----------------------------------------
You can't change the security group a running instance is using, but you can add or modify rules. If you go to "[Security Groups](https://console.aws.amazon.com/ec2/home?#s=SecurityGroups)". Click on the security group your instance is using. Go to the bottom of the page and click 'inbound'. You'll be able to add your rules there.

**Comment by osrf-migration** [comment]
**Original comment by Chad Jenkins (Bitbucket: [odestcj](https://bitbucket.org/%7B5e503a35-1f78-48c6-9010-edbd488b5cdc%7D/)).**

----------------------------------------
Thanks for the response, Stephen.  

The simulator instance was spawned through the cloudsim console, where there is no option to specify the security group before starting the instance.  I could modify simulator.py at line 226, but this seems a bit brute force.

**Comment by osrf-migration** [comment]
**Original comment by Carlos Agüero (Bitbucket: [caguero](https://bitbucket.org/%7Bda8a8e89-4bb0-421b-bd0e-dbbed3d4ed6a%7D/), GitHub: [caguero](https://github.com/caguero)).**

----------------------------------------

* changed state from "new" to "on hold"

On hold until the end of VRC.

**Comment by osrf-migration** [comment]
**Original comment by Hugo Boyer (Bitbucket: [hugomatic](https://bitbucket.org/%7B3d87874e-7099-4981-b04f-57f0272faa7d%7D/), GitHub: [hugomatic](https://github.com/hugomatic)).**

----------------------------------------

* changed state from "on hold" to "open"
